src/app/api/signup/route.ts

```diff
@@ -0,0 +1,40 @@
+import { NextRequest, NextResponse } from 'next/server';
+
+export async function POST(req: NextRequest) {
+  const { loginId, password, email, part, username, teamId } = await req.json();
+  const sendingDataObject = {
+    loginId,
+    password,
+    email,
+    part,
+    username,
+    teamId,
+  };
+
+  try {
+    const tmpResponse = await fetch(
+      'http://43.202.139.24:8080/api/user/signup',
```

Comment:
여기 api 주소 위에 정의해놓으신거로 바꿔주면 좋을 것 같아요!
```suggestion
'/api/user/signup ',